hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/table/ITTestSchemaEvolution.java

```diff
@@ -245,35 +247,44 @@ private void changeTableSchema(TableOptions tableOptions, boolean shouldCompactB
         doCompact(conf);
       }
 
-      Schema intType = SchemaBuilder.unionOf().nullType().and().intType().endUnion();
-      Schema longType = SchemaBuilder.unionOf().nullType().and().longType().endUnion();
-      Schema doubleType = SchemaBuilder.unionOf().nullType().and().doubleType().endUnion();
-      Schema stringType = SchemaBuilder.unionOf().nullType().and().stringType().endUnion();
-      Schema structType = SchemaBuilder.builder().record("new_row_col").fields()
-              .name("f0").type(longType).noDefault()
-              .name("f1").type(stringType).noDefault().endRecord();
-      Schema arrayType = Schema.createUnion(SchemaBuilder.builder().array().items(stringType), SchemaBuilder.builder().nullType());
-      Schema mapType = Schema.createUnion(SchemaBuilder.builder().map().values(stringType), SchemaBuilder.builder().nullType());
-
-      writeClient.addColumn("salary", doubleType, null, "name", AFTER);
+      // Create nullable primitive types using HoodieSchema
+      HoodieSchema intType = HoodieSchema.createNullable(HoodieSchema.create(HoodieSchemaType.INT));
```

Comment:
There is a helper method that allows you to just call `HoodieSchema.createNullable(HoodieSchemaType.INT)` for easy test setup